base_app/src/app/app.component.ts

```diff
@@ -37,7 +37,6 @@ export class AppComponent implements OnInit, AfterViewInit /*, OnChanges */ {
     _refreshRate = 1000;
     _pendingRequestWait = 10000;
     _updateSubscription = null;
-    _milliSecondsBeforeAutoPageReload = AppComponent._minutesBeforeAutoPageReload_Default * 60 * 1000;
```

Comment:
I wonder if we should keep this, but change the default value to 0 and the behavior to not ever do the page reload when the value is 0.  The argument for keeping it is it gives the flexibility to periodically auto-refresh to work around bugs that scramble the screen that we may not be aware of yet.

The argument against is that it leaves a lot of junk in the code.

Reply:
I think it'd be best to leave it out for now. Since we currently don't need it, and it's not super hard to re-implement if we do need it, I think it'd be best to remove it to keep the code neat